test/network/NetworkSettings.ts

```diff
@@ -179,13 +181,21 @@ describe('NetworkSettings', () => {
                 const reserveToken2 = await createTestToken();
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.false;
                 expect(await networkSettings.isTokenWhitelisted(reserveToken2.address)).to.be.false;
+                expect(await networkSettings.protectedTokenWhitelist()).not.to.have.members([
```

Comment:
`to.be.empty` would be a little more restrictive.

Reply:
The test only checks the difference between adding/removing specific tokens, so it's unnecessary also to assume that the list is empty (which is incidentally the case)